Simplenote/Classes/SPNoteEditorViewController.h

```diff
@@ -57,11 +52,23 @@
     
 }
 
+// Navigation Back Button
+@property (nonatomic, strong) UIButton *backButton;
+
+// Navigation Buttons
+@property (nonatomic, strong) UIButton *actionButton;
+@property (nonatomic, strong) UIButton *checklistButton;
+@property (nonatomic, strong) UIButton *keyboardButton;
+@property (nonatomic, strong) UIButton *createNoteButton;
+
 @property (nonatomic, strong) Note *currentNote;
 @property (nonatomic, strong) SPEditorTextView *noteEditorTextView;
 @property (nonatomic, strong) SPTagView *tagView;
 @property (nonatomic, strong) NSString *searchString;
-@property (nonatomic, assign) BOOL isPreviewing;
+
+@property (nonatomic, getter=isEditing) BOOL editing;
```

Comment:
Thanks for pointing this out Jorge!

I'm curious if you got a compiler error for this or not. I tried enabling a handful of different warnings but I wasn't able to generate one.

Currently I'm redeclaring but not redefining so I thought that might prevent the error. But even if I change the declaration, no error…hmm.

In my reading of the docs, the editing property on UIViewController is specific to the system provided edit button, which we don't use. So this usage doesn't look to be a conflict. Just in case, I'll rename this BOOL to avoid any unexpected side-effects though.

Reply:
No side effects, **but** this happened a while ago #20

Sometimes the compiler don't bug you now, but might start complaining, eventually. It's highly likely if this class was in Swift, you'd get such error.

(Which is something that's in the plans, swiftify all the things!)

Reply:
Yup, definitely seems like it's better to be safe than sorry!